src/components/form-helpers/form-helpers.js

```diff
@@ -320,9 +322,16 @@ const _PhoneFieldItem = ({
       autoPlaceholder: 'aggressive',
       preferredCountries: ['ar', 'mx', 'co', 'es', 'ec', 'cl', 'pe', 'us'],
       initialCountry: 'auto',
-      geoIpLookup: (callback) => {
-        // TODO: determine current country using geolocation
-        callback('ar');
+      geoIpLookup: async (success, failure) => { // TODO: read cache before request
+        try {
+          const response = await axios.get('https://ipinfo.io/json');
+          var countryCode = (response && response.country) ? response.country : "";
```

Comment:
We don't use `var` anymore. 
```suggestion
          const countryCode = (response && response.country) ? response.country : '';
```